packages/help-center/src/components/help-center-content.tsx

```diff
@@ -83,10 +83,11 @@ const HelpCenterContent: React.FC< { isRelative?: boolean; currentRoute?: string
 	);
 
 	useEffect( () => {
-		if ( navigateToRoute ) {
+		if ( navigateToRoute && location.pathname !== navigateToRoute ) {
```

Comment:
Is `location.pathname !== navigateToRoute` to don't navigate multiple times to the same article?
I think we may need to change this, I tried logging and when clicking on the same article the second time:


navigateToRoute = `/post/?link=https%3A%2F%2Fit.support.wordpress.com%2Fguida-alle-decisioni-per-la-costruzione-del-sito-web-della-membership%2F&postId=2718&blogId=150300509&cacheBuster=1722607102098``

location.pathname = `/post`

So it keeps navigating there

Reply:
Yes. And you're right!

Reply:
Addressed